ui/apps/platform/src/Containers/Vulnerabilities/types.ts

```diff
@@ -81,3 +81,18 @@ export type ObservedCveMode = (typeof observedCveModeValues)[number];
 export function isObservedCveMode(value: unknown): value is ObservedCveMode {
     return observedCveModeValues.some((mode) => mode === value);
 }
+
+export type VerifiedStatus =
+    | 'CORRUPTED_SIGNATURE'
+    | 'FAILED_VERIFICATION'
+    | 'GENERIC_ERROR'
+    | 'INVALID_SIGNATURE_ALGO'
+    | 'UNSET'
+    | 'VERIFIED';
+
+export type SignatureVerificationResult = {
+    status: VerifiedStatus;
+    verificationTime: string;
```

Comment:
Elsewhere, I have added `// ISO 8601 formatted date` as comment at the end of the line

Reply:
Wouldn't that technically be `date time`? Anyways added a comment.